### ISSUE TITLE
Cannot use multiple python io plugins at the same time

### ISSUE THREAD

**Issue by condret** [radareorg/radare2-rlang#25]
the corresponding wrapper uses global scope vars to store the plugin's python function pointers

**Comment by trufae** [comment]
thats not true, you can open many python plugins and they all work at the same time, they just run in the same single python vm. is this the issue you are trying to solve?
